src/lib/Event/Listener/LoginListener.php

```diff
@@ -100,15 +98,20 @@ public function onSecurityInteractiveLogin(InteractiveLoginEvent $event): void
         }
     }
 
+    private function getCustomerId(): string
+    {
+        return (string) $this->configResolver->getParameter(
+            'authentication.customer_id',
+            Parameters::NAMESPACE
+        );
```

Comment:
From task description: 
> Expected:
> Login notification to reco service should only be send, when customer_id is set for the current Site Access

If customer id should be taken for current siteAccess then we need to inject `SiteAccessService` to get current site access and pass it name as third argument `$scope` to `getParameter` method.